pkg/view/component/link.go

```diff
@@ -95,6 +106,11 @@ func (t *Link) SetStatus(status TextStatus, detail Component) {
 	t.Config.StatusDetail = detail
 }
 
+// SetComponent sets the component to be wrapped by the Link
```

Comment:
Let's also add a `WithComponent` that can be passed as a `LinkOption`

Reply:
Added at line https://github.com/vmware-tanzu/octant/pull/2255/files#diff-e4b9f4aec321f580471b37d920edfa5164cc21ea234e4ff7395f96926480bc64R139

Here is an example of how to use it:

```
btn := component.NewButton("single button in link", action.Payload{})
btnLink := component.NewLink("title", "value", requestPath, component.WithComponent(btn))
```